theme/utils.py

```diff
@@ -99,16 +99,7 @@ def get_user_profiles_missing_bucket_name(usernames=None):
     :return a tuple of two lists: a list of users and a list of UserProfile instances
     """
     from theme.models import UserProfile
-    bad_ups = []
     ups = UserProfile.objects.filter(_bucket_name__isnull=True).filter(user__is_active=True)
     if usernames:
         ups = ups.filter(user__username__in=usernames)
-    for up in ups:
-        u = up.user
-        resources = u.uaccess.owned_resources
```

Comment:
okay so you don't have to own a resource in order to be flagged as a "bad_up" if you don't have a bucket name. Cool